### PR TITLE
feat: remove plugin-related contracts

### PR DESCRIPTION
**Pull request by Michael-A-Heuer**
## Description

This PR removes the 
- [x] `Admin`, `AdminSetup` contracts that have been moved to [aragon/admin-plugin](https://github.com/aragon/admin-plugin) (see https://github.com/aragon/admin-plugin/pull/1)
- [x] `Multisig`, `MultisigSetup` contracts that have been moved to [aragon/multisig-plugin](https://github.com/aragon/admin-plugin) (see https://github.com/aragon/admin-plugin/pull/1)
- [x] `TokenVoting`, `TokenVotingSetup` contracts have been moved to [aragon/token-voting-plugin](https://github.com/aragon/token-voting-plugin) (see https://github.com/aragon/token-voting-plugin/pull/1)
- [x] `AddresslistVoting`, `AddresslistVotingSetup` contracts will be deprecated

and related metadata.

Task ID: [OS-489](https://aragonassociation.atlassian.net/browse/OS-489)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [x] I have updated the `CHANGELOG.md` file in the root folder.
- [ ] I have updated the `DEPLOYMENT_CHECKLIST` file in the root folder.
- [ ] I have updated the `UPDATE_CHECKLIST` file in the root folder.

Adapting the checklists will be done as part of [OS-924](https://aragonassociation.atlassian.net/browse/OS-924).

[OS-489]: https://aragonassociation.atlassian.net/browse/OS-489?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[OS-924]: https://aragonassociation.atlassian.net/browse/OS-924?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/contracts/test/framework/dao/dao-factory.ts [hunk 1]
```diff
@@ -533,208 +515,4 @@ describe('DAOFactory: ', function () {
 
     expect(installationAppliedEventCount).to.equal(2);
   });
-
-  describe('E2E: Install,Update,Uninstall Plugin through Admin Plugin', async () => {
```
Comment (Michael-A-Heuer) [inline packages/contracts/test/framework/dao/dao-factory.ts:537]: We test the plugin setup processing extensively in the three GH repos.

File: packages/contracts/test/deploy/updateTo1_3_0.ts [hunk 2]
```diff
@@ -1,49 +0,0 @@
...
-
-      await initForkForOsxVersion(network, previousOsxVersion);
-
-      const updateDeployTags = ['v1.3.0'];
-      await initializeDeploymentFixture(updateDeployTags);
-    });
-
-    it('deploys new contracts with new addresses', async function () {
-      const changedContracts = [
-        'DAOFactory',
```
Comment (jordaniza) [inline packages/contracts/test/deploy/updateTo1_3_0.ts:30]: Why is this deleted with the DAOFactory in it

Comment (Michael-A-Heuer) [reply packages/contracts/test/deploy/updateTo1_3_0.ts:30]: We won't do the update to 1.3.0 anymore and the "tests" in here aren't really testing anything.
We just check that some contracts were deployed and 4/5 we don't deploy anymore.
We should think about better deployment tests in the future.

Comment (jordaniza) [reply packages/contracts/test/deploy/updateTo1_3_0.ts:30]: fine then leave it, I just wanted to check

**Review by novaknole — APPROVED** [review]
